dev-python/elasticsearch-py/elasticsearch-py-7.7.0.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2020 Gentoo Authors
```

Comment:
Don't  change PYTHON_COMPAT on stable versions.

Reply:
No, it's okay to do that.